packages/storybook-builder/src/rollup-plugin-prebundle-modules.ts

```diff
@@ -0,0 +1,190 @@
+    try {
+      require.resolve(id, { paths: [process.cwd()] });
+      return true;
+    } catch (e) {
+      return false;
+    }
+  });
+  return include;
+}
+
+// taken from https://github.com/storybookjs/storybook/blob/v7.0.9/code/lib/builder-vite/src/optimizeDeps.ts#L7
+export const CANDIDATES = [
+  '@base2/pretty-print-object',
+  '@emotion/core',
+  '@emotion/is-prop-valid',
+  '@emotion/styled',
+  '@mdx-js/react',
+  '@storybook/addon-docs > acorn-jsx',
+  '@storybook/addon-docs',
+  '@storybook/addon-essentials/docs/mdx-react-shim',
+  '@storybook/channel-postmessage',
+  '@storybook/channel-websocket',
+  '@storybook/client-api',
+  '@storybook/client-logger',
+  '@storybook/core/client',
+  '@storybook/global',
+  '@storybook/preview-api',
+  '@storybook/preview-web',
+  '@storybook/react > acorn-jsx',
+  '@storybook/react',
+  '@storybook/svelte',
+  '@storybook/types',
+  '@storybook/vue3',
+  'acorn-jsx',
+  'acorn-walk',
+  'acorn',
+  'airbnb-js-shims',
+  'ansi-to-html',
+  'axe-core',
+  'color-convert',
+  'deep-object-diff',
+  'doctrine',
+  'emotion-theming',
+  'escodegen',
+  'estraverse',
+  'fast-deep-equal',
+  'html-tags',
+  'isobject',
+  'jest-mock',
+  'loader-utils',
+  'lodash/camelCase.js',
+  'lodash/camelCase',
+  'lodash/cloneDeep.js',
+  'lodash/cloneDeep',
+  'lodash/countBy.js',
+  'lodash/countBy',
+  'lodash/debounce.js',
+  'lodash/debounce',
+  'lodash/isEqual.js',
+  'lodash/isEqual',
+  'lodash/isFunction.js',
+  'lodash/isFunction',
+  'lodash/isPlainObject.js',
+  'lodash/isPlainObject',
+  'lodash/isString.js',
+  'lodash/isString',
+  'lodash/kebabCase.js',
+  'lodash/kebabCase',
+  'lodash/mapKeys.js',
+  'lodash/mapKeys',
+  'lodash/mapValues.js',
+  'lodash/mapValues',
+  'lodash/merge.js',
+  'lodash/merge',
+  'lodash/mergeWith.js',
+  'lodash/mergeWith',
+  'lodash/pick.js',
+  'lodash/pick',
+  'lodash/pickBy.js',
+  'lodash/pickBy',
+  'lodash/startCase.js',
+  'lodash/startCase',
+  'lodash/throttle.js',
+  'lodash/throttle',
+  'lodash/uniq.js',
+  'lodash/uniq',
+  'lodash/upperFirst.js',
+  'lodash/upperFirst',
+  'markdown-to-jsx',
+  'memoizerific',
+  'overlayscrollbars',
+  'polished',
+  'prettier/parser-babel',
+  'prettier/parser-flow',
+  'prettier/parser-typescript',
+  'prop-types',
+  'qs',
+  'react-dom',
+  'react-dom/client',
+  'react-fast-compare',
+  'react-is',
+  'react-textarea-autosize',
+  'react',
+  'react/jsx-runtime',
+  'refractor/core',
+  'refractor/lang/bash.js',
+  'refractor/lang/css.js',
+  'refractor/lang/graphql.js',
+  'refractor/lang/js-extras.js',
+  'refractor/lang/json.js',
+  'refractor/lang/jsx.js',
+  'refractor/lang/markdown.js',
+  'refractor/lang/markup.js',
+  'refractor/lang/tsx.js',
+  'refractor/lang/typescript.js',
+  'refractor/lang/yaml.js',
+  'regenerator-runtime/runtime.js',
+  'slash',
+  'store2',
+  'synchronous-promise',
+  'telejson',
+  'ts-dedent',
+  'unfetch',
+  'util-deprecate',
+  'vue',
+  'warning',
```

Comment:
This is the list that was supposed to have shrunk already right? Or is this a different list?

Reply:
it's the old one
new one I have in a local branch only and needs more testing
I'd prefer to work on it after finishing this

Reply:
fine by me 👍  just checking